src/components/Board.js

```diff
@@ -5,29 +5,89 @@ import axios from 'axios';
 import './Board.css';
 import Card from './Card';
 import NewCardForm from './NewCardForm';
-import CARD_DATA from '../data/card-data.json';
 
 class Board extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
 
     this.state = {
       cards: [],
     };
   }
 
+  componentDidMount() {
+    console.log("Component has mounted");
+    const CARDS = this.props.url + this.props.boardName
+
+    axios.get(CARDS)
+    .then((response) => {
+      this.setState({
+        cards: response.data,
+      });
+    })
+    .catch((error) => {
+      this.setState({
+        error: error.message
+      });
+    });
+  }
+
+  addCardCallback = (cardInfo) => {
+    // console.log("Board message - addCardCallback triggered");
+    axios.post('https://inspiration-board.herokuapp.com/boards/sig/cards', cardInfo)
```

Comment:
Hm, it'd be nice if you could have used the props `boardName` and `url` with this too